source/manual/covid-19-services.html.md

```diff
@@ -155,6 +155,12 @@ All applications are Rails applications, and use the
 about [managing PaaS applications][paas-managing] and [monitoring PaaS
 applications][paas-monitoring].
 
+#### Manage deployment secrets
+
+Secrets are configured using Reliability Engineering's secrets management
+system.  [Detailed documentation][re-secrets] is available in the
```

Comment:
[One space please](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style) 😄 

```suggestion
system. [Detailed documentation][re-secrets] is available in the
```

Reply:
This goes anything everything I was ever taught .... but it's in the style guide, so I've made the change 😁